addon/services/apollo.js

```diff
@@ -137,7 +137,7 @@ export default class ApolloService extends Service {
     let config = getOwner(this).resolveRegistration('config:environment');
     if (config && config.apollo) {
       return config.apollo;
-    } else if (Ember.testing) {
+    } else if (isTesting()) {
```

Comment:
This could (should?) be combined with `macroCondition(...)`, so the testing code is stripped out when `isTesting()` is `false`.